### PR TITLE
update UG and UG

### PR DESCRIPTION
**Pull request by Cao-Zeyu**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -457,29 +457,12 @@ borrowed books tracking, and module-related functions. This increase users' effi
...
-## Instructions for manual testing
-
 ### Launch and shutdown
 1. Initial launch
-    1. Download the jar file and copy to an empty folder.
-    2. Open a command line window in the same directory and type `java -jar termiNus.jar` to launch.
+    1. Download the latest version of `termiNus` from [here](https://github.com/AY2021S1-CS2113-T14-3/tp/releases/latest) 
+    and copy the jar file to an empty folder.
+    2. Open a command line window in the same directory and launch termiNus by typing `java -jar termiNus.jar` and press enter.
 2. Shutdown
```
Comment (MuhammadHoze) [inline docs/DeveloperGuide.md]: For this part, u can modify the pointers to this : 
image

Comment (MuhammadHoze) [reply docs/DeveloperGuide.md]: Do include the `<br>` if not they will be all on the same line

Comment (Cao-Zeyu) [reply docs/DeveloperGuide.md]: Sure!

**Comment by GuoAi** [comment]
LGTM!

**Review by MuhammadHoze — APPROVED** [review]

**Pull request merged**